src/Model.h

```diff
@@ -110,6 +121,8 @@ struct PopVar
 	int cumItype[INFECT_TYPE_MASK], cumI_keyworker[2], cumC_keyworker[2], cumT_keyworker[2];
 	Infection *inf_queue[MAX_NUM_THREADS]; // the queue (i.e. list) of infections. 1st index is thread, 2nd is person.
 	int n_queue[MAX_NUM_THREADS]; 	// number of infections in inf_queue
+	HostClosure *host_closure_queue;  // When places close, host absenteeism times get set within threaded loop.
+	int host_closure_queue_size; // Number of hosts in a timestep to set min/max absenteeism times for. 
```

Comment:
I think it would be good to explicitly reference the array that this gives the size of, i.e. `Number of host closures in host_closure_queue.`.

Reply:
Done!